### ISSUE TITLE
Status page not connecting

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-kubernetes#50]
When I create a fresh cluster I find that the status page does not show up.  I'm not sure how to get more information to diagnose this.

**Comment by mrocklin** [comment]
Nothing showed up in the logs

```
distributed.scheduler - INFO - -----------------------------------------------
distributed.scheduler - INFO -   Scheduler at:      tcp://10.24.0.3:8786
distributed.scheduler - INFO -        http at:                     :9786
distributed.scheduler - INFO -       bokeh at:                     :8787
distributed.scheduler - INFO - Local Directory:    /tmp/scheduler-hgsq3zku
distributed.scheduler - INFO - -----------------------------------------------
distributed.scheduler - INFO - Register tcp://10.24.0.4:38917
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.0.4:38917
distributed.scheduler - INFO - Register tcp://10.24.0.5:37957
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.0.5:37957
distributed.scheduler - INFO - Register tcp://10.24.2.3:42503
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.2.3:42503
distributed.scheduler - INFO - Register tcp://10.24.2.5:44721
distributed.scheduler - INFO - Register tcp://10.24.2.4:33198
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.2.4:33198
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.2.5:44721
distributed.scheduler - INFO - Register tcp://10.24.1.5:41392
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.1.5:41392
distributed.scheduler - INFO - Register tcp://10.24.1.4:38191
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.1.4:38191
distributed.scheduler - INFO - Register tcp://10.24.1.3:39715
distributed.scheduler - INFO - Starting worker compute stream, tcp://10.24.1.3:39715
distributed.scheduler - INFO - Receive client connection: Client-f4bd3f54-a186-11e7-8014-0a580a180002
distributed.scheduler - INFO - Connection to client Client-f4bd3f54-a186-11e7-8014-0a580a180002 broken
distributed.scheduler - INFO - Remove client Client-f4bd3f54-a186-11e7-8014-0a580a180002
distributed.scheduler - INFO - Close client connection: Client-f4bd3f54-a186-11e7-8014-0a580a180002
distributed.scheduler - INFO - Receive client connection: Client-a49c2a1e-a187-11e7-8070-0a580a180002
distributed.scheduler - INFO - Connection to client Client-a49c2a1e-a187-11e7-8070-0a580a180002 broken
distributed.scheduler - INFO - Remove client Client-a49c2a1e-a187-11e7-8070-0a580a180002
```

**Comment by mrocklin** [comment]
Ah, ok.  Somehow this pulled in bokeh 0.12.4 rather than the latest version

**Comment by martindurant** [comment]
Fixed in #51 , then?

**Comment by mrocklin** [comment]
u
